hi-ml-cpath/src/health_cpath/utils/heatmap_utils.py

```diff
@@ -9,23 +9,25 @@
 
 def location_selected_tiles(tile_coords: np.ndarray,
                             location_bbox: List[int],
-                            level: int) -> np.ndarray:
+                            scale_factor: int = 1,
+                            should_upsacle_coords: bool = True) -> np.ndarray:
     """ Return the scaled and shifted tile co-ordinates for selected tiles in the slide.
     :param tile_coords: XY tile coordinates, assumed to be spaced by multiples of `tile_size`
```

Comment:
nitpick: docu is not rendered correctly if there is no blank linebefore `:param`